src/client/vue/pages/PagePlayRemote.vue

```diff
@@ -245,6 +245,24 @@ const initGameView = async () => {
     }
 };
 
+const makeTitle = (gameClient: HostedGameClient) => {
+    const players = gameClient.getPlayers();
+    const { state } = gameClient.getHostedGame();
+    if (players.length < 2 && 'created' === state)
+        return `Waiting for an opponent: hosted by ${players[0].pseudo}`;
+    const pairing = players.map(p => p.pseudo).join(' VS ');
+    const capitalizedState = state[0].toUpperCase() + state.slice(1);
+    let yourTurn = '';
+    if (loggedInPlayer.value != null) {
+        const player = loggedInPlayer.value;
+        const index = players.findIndex(p => p.publicId === player.publicId);
+        if (index != null && gameClient.getGame().getCurrentPlayerIndex() === index) {
+            yourTurn = '• Your turn! • ';
+        }
+    }
+    return `${yourTurn}${capitalizedState}: ${pairing}`;
```

Comment:
To prevent reusing state string which is not meant to be showed on UI, and doing capitalize, you can use translations:

Adding in `en.json`:

```
    "game_state": {
        "playing": "Playing",
        "canceled": "Canceled",
        ...
    },
```

Then:

``` diff
+import i18next from 'i18next';
```

```suggestion
    return `${yourTurn}${i18next.t('game_state.' + state)}: ${pairing}`;
```

Reply:
Added (although this part wasn't changed in this PR)